app/helpers/test_executions_helper.rb

```diff
@@ -101,8 +101,8 @@ def execution_status_class(execution)
 
   def info_title_for_product_test(product_test)
     case product_test
-    when MeasureTest then 'Measure Test Information'
-    when FilteringTest then 'Filtering Test Information'
+    when MeasureTest then "#{product_test.cms_id} Measure Test"
+    when FilteringTest then "#{product_test.cms_id} Filtering Test"
     when ChecklistTest then 'Record Sample Test Information'
     else 'Test Information'
```

Comment:
May be considered out of scope, but we should potentially have a clearer title for MultiMeasureTests (EH Test, EP or EC Test), which currently end up as the less informative 'Test Information' category.